acceptance-tests/run-tests

```diff
@@ -18,12 +18,17 @@ setArgsOverrides(
     .option('accountId', {
       alias: ['p', 'portalId', 'a'],
       type: 'string',
-      description: 'Account ID',
+      description: 'The account ID to use',
     })
     .option('cliPath', {
       alias: 'c',
       type: 'string',
-      description: 'CLI path',
+      description: 'The path to the CLI',
+    })
+    .option('cliNPMVersion', {
+      alias: 'npm',
+      type: "string",
+      description: "The npm version of the CLI to download and run"
```

Comment:
Might make sense to say that specifying a version will cause the tests to run with `npx`, rather than "download and run" (even though npx technically does download"